starter-core/src/main/java/io/micronaut/starter/feature/test/MicronautRestAssuredValidator.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2017-2022 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.starter.feature.test;
+
+import io.micronaut.starter.application.ApplicationType;
+import io.micronaut.starter.feature.Feature;
+import io.micronaut.starter.feature.validation.FeatureValidator;
+import io.micronaut.starter.options.Options;
+import jakarta.inject.Singleton;
+
+import java.util.Set;
+
+@Singleton
+public class MicronautRestAssuredValidator implements FeatureValidator {
```

Comment:
remove `MicronautRestAssuredValidator`. 

You can use `micronaut-test-rest-assured` with Spock and KoTest. 

> You can then inject instances of [RequestSpecification](https://www.javadoc.io/static/io.rest-assured/rest-assured/5.1.0/io/restassured/specification/RequestSpecification.html) into test fields or method parameters 
(parameters are only supported with JUnit 5):

Based on the docs, I think you should be able to do: 

```groovy
@MicronautTest 
public class RestAssuredHelloWorldSpec extends Specification {
    @Inject  
   RequestSpecification requestSpec
   void "test hello world"() {
```